examples/with-apollo-auth/lib/withData.js

```diff
@@ -54,7 +54,13 @@ export default ComposedComponent => {
             <ComposedComponent url={url} {...composedInitialProps} />
           </ApolloProvider>
         )
-        await getDataFromTree(app)
+        await getDataFromTree(app, {
+          router: {
```

Comment:
Does this break when you don't wrap withData in a withRouter?

Reply:
No, it does not break, I have various pages/components, some wrapped, some not wrapped.

Reply:
Cool, thanks 👍